apps/e2e/ag-grid-storybook/src/app/data-manager/data-manager.component.spec.ts

```diff
@@ -9,6 +11,10 @@ describe('DataManagerComponent', () => {
   let fixture: ComponentFixture<DataManagerComponent>;
 
   beforeEach(() => {
+    jest
+      .spyOn(FontFaceObserver.prototype, 'load')
+      .mockImplementation(() => Promise.resolve());
```

Comment:
Instead of leaking the implementation details of the font loader service, could you create a `FontLoadingTestModule` in the shared folder that would provide a mock service automatically? Then, the consuming test would just need to import the module into its `TestBed`.
```
@NgModule({
  providers: [
    {
      provide: FontLoadingService, 
      useClass: FontLoadingTestService // or something similar?
    }
  ]
})
export class FontLoadingTestModule {}
```

Reply:
Good call. Added a testing service in a testing module.